cpu/kinetis_common/mcg.c

```diff
@@ -153,7 +153,7 @@ static void kinetis_mcg_enable_osc(void)
         MCG->C2 = (uint8_t)(MCG_C2_RANGE0(0));
     }
 
-    OSC0->CR = (uint8_t)(OSC_CR_ERCLKEN_MASK | OSC_CR_EREFSTEN_MASK
+    OSC->CR = (uint8_t)(OSC_CR_ERCLKEN_MASK | OSC_CR_EREFSTEN_MASK
```

Comment:
Please do not change cpu/kinetis_common/mcg.c  here (if there is something wrong in kinetis_common then open a PR for it)